hrui/hrui.php

```diff
@@ -594,3 +615,51 @@ function hrui_civicrm_alterContent( &$content, $context, $tplName, &$object ) {
    </script>";
   }
 }
+
+/**
+ * @param int $contractID contract ID
+ * @return string|null return an comma separated string containing the list of departments
+ * for the specific contact based on current contract ID
+ */
+function getDepartmentsList($contractID)  {
```

Comment:
It looks more like this method is retrieving a list of departments for a job roles, so maybe it would be better to place it in the HrJobRole BAO